src/NewTwitchApi/Resources/EventSubApi.php

```diff
@@ -0,0 +1,286 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NewTwitchApi\Resources;
+
+use GuzzleHttp\Exception\GuzzleException;
+
+class EventSubApi extends AbstractResource
+{
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/eventsub
+     */
+    private function subscribe(string $type, string $version, array $condition, string $callback, string $bearer, string $secret): void
```

Comment:
- Bearer should always be the first parameter
- Since Secrets are required for all Subscriptions, put this directly after the Bearer
- Since Callbacks are required for all Subscriptions, put this after the Secret
- Type, Version and Condition to follow